configure.ac

```diff
@@ -0,0 +1,19 @@
+#                                               -*- Autoconf -*-
```

Comment:
What else would you recommend?—
Alex Striff

On Mon, Jan 12, 2015 at 7:35 AM, Augustin Cavalier
<notifications@github.com> wrote:

>> @@ -0,0 +1,19 @@
>> +#                                               -*- Autoconf -*-
> Why are you using autoconf?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/HaikuArchives/AVLDupTree/pull/1/files#r22797159